### PR TITLE
update npm dependencies

### PR DESCRIPTION
**Pull request by larshp**

### CODE REVIEW

File: test/cl_run.clas.abap [hunk 1]
```diff
@@ -40,7 +41,8 @@ CLASS cl_run IMPLEMENTATION.
       ENDIF.
     ENDLOOP.
 
-    CONCATENATE LINES OF string_tab INTO result SEPARATED BY |\n|.
+    newline = |\n|.
+    CONCATENATE LINES OF string_tab INTO result SEPARATED BY newline.
```
Comment (schneidermic0) [inline test/cl_run.clas.abap:45]: Is this changer needed? Is transpiler facing issues with the old syntax?

Comment (larshp) [reply test/cl_run.clas.abap:45]: yea, it was not valid ABAP syntax

Comment (schneidermic0) [reply test/cl_run.clas.abap:45]: I see, you are right.

**Review by schneidermic0 — APPROVED** [review]
LGTM. Thanks, @larshp.

**Pull request merged**